handbook/marketing/adding_screenshots_screen_recording.md

```diff
@@ -0,0 +1,116 @@
+
+## Adding a screen recording (screencast)
+
+A screen recording (screencast) is a great way to see a feature or flow in action, and we want everyone at Sourcegraph to feel capable of creating GIFs and/or videos. This doesn’t mean though that you’re on your own, so if you need assistance of any kind, including something created entirely for you, just reach out to your mates on the marketing team and we’ll help you get it sorted!
+
+### Screen recording apps
+
+The reason to use a paid tool is more settings, export quality controls, and a nicer workflow for producing assets. If you’re planning on doing this more than once, it’s worth investing in a professional tool.
+
+- [Cleanshot](https://getcleanshot.com/)<br/>
+Export screen recordings to GIF or video and nice features such as hiding desktop icons and notifications during recordings automatically. It does not have any editing or annotation tools for screen recordings though.
+
+- [Quicktime (videos only)](https://support.apple.com/en-au/guide/quicktime-player/qtp97b08e666/mac)<br/>
+Can capture a window, a selection of, or the entire screen, and in conjunction with the "Split Clip" functionality makes trimming the start and end easy. It’s a great tool that’s free on macOS.
+
+- [ScreenFlow](https://www.telestream.net/screenflow/overview.htm)<br/>
+Simply the best screen recording, editing, and production tool available. Designed primarily for video work (e.g. editing) but can export to GIF.
+
+### Screen recording tips
+
+Recording a great screencast is a large topic, but here are some essentials to keep in mind:
+
+- Ensure it has an easy to demonstrate and clear purpose, e.g. show smart search field autocomplete, syntax highlighting, and error handling
+- Make it as short and simple as possible, but not so short, it’s hard to know what you’re demonstrating
+- Only capture the part of the screen that is necessary. If the output is video, then aim for 16:9 ratio for YouTube
+- If the entire screen needs to be captured, then your resolution should be 1280x720 (720p), as that will make on-screen elements easy to read
+- Decide whether it should be standalone, or if it only makes sense when embedded with content
+- Ideally, GIFs should less than 15-20 seconds in length, otherwise, it’s most likely better as a video
+- Invest in learning a tool such as ScreenFlow if you need to annotate and/or edit your recordings
+- Does your GIF or video make sense without an accompanying explanation? If not, then a VoiceOver or text annotations can help a lot
+
+Like most things, practice makes perfect, but if you’re wanting to get some advice before taking the plunge, Ryan, Thorsten, and Quinn (sqs), have all created screencasts and can help you get started, so just reach out on Slack.
+
```

Comment:
A brief section on audio would be really helpful (I've just done it at the same time, but then can't do the nice speed-ups in different sections). 

```suggestion

### Adding audio or voice-overs

TODO

```